x/blob/types/errors.go

```diff
@@ -33,5 +33,7 @@ var (
 	ErrNoShareCommitments             = errors.Register(ModuleName, 11135, "no share commitments provided")
 	ErrInvalidNamespace               = errors.Register(ModuleName, 11136, "invalid namespace")
 	ErrInvalidNamespaceVersion        = errors.Register(ModuleName, 11137, "invalid namespace version")
-	ErrTotalBlobSizeTooLarge          = errors.Register(ModuleName, 11138, "total blob size too large")
+	// ErrTotalBlobSize is deprecated, use ErrBlobTooLarge instead.
+	ErrTotalBlobSizeTooLarge = errors.Register(ModuleName, 11138, "total blob size too large")
+	ErrBlobTooLarge          = errors.Register(ModuleName, 11139, "blob(s) too large")
```

Comment:
[nit-suggestion]  
```suggestion
	ErrBlobsTooLarge          = errors.Register(ModuleName, 11139, "blob(s) too large")
```

Reply:
Updated in 0774127b5058d1610413aedc16ec21df87ca44ca